lib/poetry-lock/poetry-lock-all.sh

```diff
@@ -27,15 +27,15 @@ for i in ${tomls}; do
         (
             echo "--------------------- special casing in $i"
             cd $(dirname "$i")
-            poetry lock --no-update || fail "broke on special case 'poetry lock --no-update' for $i"
+            poetry lock || fail "broke on special case 'poetry lock' for $i"
```

Comment:
We should require poetry 2.0 here.  2.0 makes no-update the default, old poetry would change a lot more stuff.